### DISCUSSION TITLE
Does react-query has selectors feature?

### DISCUSSION THREAD

**Discussion by tranhaminh91** [TanStack/query#1223] [Q&A]
I use react-query to fetch a list of posts and want to filter only the completed post.

I would like the selectors feature like recoil, for example:

https://recoiljs.org/docs/basic-tutorial/selectors/

**Comment by TkDodo** [comment]
it's coming in V3: https://react-query-beta.tanstack.com/guides/migrating-to-react-query-3#selectors
